pydis_site/templates/events/pages/code-jams/9/frameworks.html

```diff
@@ -17,6 +17,16 @@
         If there is a library not listed below that you think should be here, you're welcome to discuss it with the Events Team over at <a href="https://discord.gg/HnGd3znxhJ">the server</a>.
     </p>
 
+    <div class="notification is-info is-light">
+        <p>Most of the below frameworks implement what is called the ASGI Specification.
+            This specification documents how the frameworks should interact with ASGI servers and call the frameworks.
```

Comment:
```suggestion
            This specification documents how the frameworks should interact with ASGI servers.
```
This reads better to me personally, thoughts?

Reply:
I agree, commited! 👍